ccm_web/server/src/user/user.model.ts

```diff
@@ -0,0 +1,44 @@
+import { Column, Model, Table, Unique } from 'sequelize-typescript'
+import { Optional, DataTypes } from 'sequelize'
+
+// https://www.npmjs.com/package/sequelize-typescript#more-strict
+
+interface UserAttributes {
+  id?: number
+  loginId: string
+  firstName: string
+  lastName: string
+  email: string
+}
+
+interface UserCreationAttributes extends Optional<UserAttributes, 'id'> {
+  createdAt?: Date
+  updatedAt?: Date
+}
+
+// Sequelize Datatypes mapping with Mysql https://sequelize.org/master/manual/model-basics.html#data-types
+@Table({
+  freezeTableName: true
+})
+export class User extends Model<UserAttributes, UserCreationAttributes> {
+  @Column({
+    type: DataTypes.STRING
+  })
+  firstName!: string
```

Comment:
it is throwing a type script error, so I need to have that assertions there

![Screen Shot 2021-05-24 at 3 31 12 PM](https://user-images.githubusercontent.com/8579775/119398216-4351eb80-bca5-11eb-8f06-fa113ed6dab0.png)

Reply:
The Author of `sequelize-typescript`, put together an example for usage of this library and he is using the assertion as [well](https://github.com/RobinBuschmann/sequelize-typescript-example/blob/master/lib/models/Actor.ts).

Reply:
Okay, seems fine. This looks to be the related issue: https://github.com/RobinBuschmann/sequelize-typescript/issues/292

This is the TypeScript feature in play I guess: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#definite-assignment-assertions

Reply:
I will resolve this now, since author suggested `!` [having](https://github.com/RobinBuschmann/sequelize-typescript/issues/292#issuecomment-386799189) it